### PR TITLE
Update README cloning instructions to set core.symlinks for Windows

### PR DESCRIPTION
**Pull request by yourslab**
Description
-----------

Because some submodules have symlinks from CBMC, copying a directory in Windows may cause a hangup. This updates the instructions to set `core.symlinks` to true for Windows.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,6 +12,11 @@ This repo uses [Git Submodules](https://git-scm.com/book/en/v2/Git-Tools-Submodu
 
 **Note:** If you download the ZIP file provided by the GitHub UI, you will not get the contents of the submodules. (The ZIP file is also not a valid git repository)
 
+If using Windows, set `core.symlinks` to true:
```
Comment (aggarw13) [inline README.md:15]: A comment about the purpose of setting this configuration will be helpful

Comment (yourslab) [reply README.md:15]: Added

**Review by aggarg — APPROVED** [review]

**Review by aggarw13 — APPROVED** [review]

**Pull request merged**